### PR TITLE
docs: add suffix example for timestamp gauge

### PR DESCRIPTION
**Pull request by naseemkullah**
markdown was also auto-formatted by IDE

### CODE REVIEW

File: content/docs/practices/naming.md [hunk 1]
```diff
@@ -7,41 +7,43 @@ sort_rank: 1
...
+- ...should use base units (e.g. seconds, bytes, meters - not milliseconds, megabytes, kilometers). See below for a list of base units.
+- ...should have a suffix describing the unit, in plural form. Note that an accumulating count has `total` as a suffix, in addition to the unit if applicable.
+- <code>http_request_duration\_<b>seconds</b></code>
+- <code>node_memory_usage\_<b>bytes</b></code>
+- <code>http_requests\_<b>total</b></code>
+  (for a unit-less accumulating count)
+- <code>process_cpu\_<b>seconds_total</b></code>
+  (for an accumulating count with unit)
+- <code>record_processed\_<b>latest_unix_time</b></code>
+  (for a timestamp that tracks a latest occurence)
```
Comment (naseemkullah) [inline content/docs/practices/naming.md:39]: here is the new entry,

debate open as to wether to standardize on `_timestamp_seconds` or `_unix_time`

unix time does imply seconds https://en.wikipedia.org/wiki/Unix_time
though the former is more explicit

**Comment by roidelapluie** [comment]
Can you remove the unrelated changes? it makes the PR hard to review, even with ignoring whitespace. We use _timestamp-seconds currently. That's what should be documented.

**Comment by naseemkullah** [comment]
> Can you remove the unrelated changes? it makes the PR hard to review, even with ignoring whitespace. We use _timestamp-seconds currently. That's what should be documented.

ca y est

**Comment by RichiH** [comment]
Thanks!

**Comment by naseemkullah** [comment]
> Thanks!

My pleasure!

**Pull request merged**